### PR TITLE
Intro and confirmation page updates

### PR DESCRIPTION
**Pull request by cehsu**
Introduction page

localhost_3001_disability-benefits_526_apply-for-increase_introduction 10

Confirmation page

localhost_3001_disability-benefits_526_apply-for-increase_confirmation 3

### CODE REVIEW

**Comment by peggygannon** [comment]
@cehsu Hi, Claire. 
I input my content edits and this PR is ready to take a screenshot so I can check the changes. 

Please note, that the SIP progress message that appears at the top of the Intro Page is not in this PR. And I still need to make an edit to that component.

**Review by peggygannon — COMMENTED** [review]
@cehsu Thanks, Claire, for posting the screenshots. 
On the introduction page, we need to update the URL for the supplemental forms page. I didn't see the URL in the copy, but the URL should be: 
[Learn what additional forms you may need to file with your disability claim](disability-benefits/apply/supplemental-forms/).

**Review by peggygannon — CHANGES_REQUESTED** [review]
Hi, Claire, see below for my edits: 

Intro page: 
Please put this sentence on its own line: 
[Learn what additional forms you may need to file with your disability claim]. 

Confirmation page:
Please add period after this sentence: 
**Your claim has been submitted.**

Confirmation page
-Remove this sentence:
If you didn't upload this form to your application, you can download it here and mail it in.

Put this sentence on its own line:
[Download VA Form 21-4142].

**Comment by cehsu** [comment]
Thanks @peggygannon kindly refer to these updated screenshots:
Introduction page:
localhost_3001_disability-benefits_526_apply-for-increase_ 4

Confirmation page:
localhost_3001_disability-benefits_526_apply-for-increase_confirmation 4

Please also note that I added a period at the end of "Check the status of your disability claim.", to be consistent with the sentence below: "Learn more about what happens after you file a disability claim." Does this work or should that be changed back?

**Review by cvalarida — APPROVED** [review]
Looks like it's good to go as soon as @peggygannon gives the :+1:
Just had a question about the TODOs left in here.

File: src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -62,18 +63,18 @@ class IntroductionPage extends React.Component {
...
-                <li>Private medical and hospital records that show your claimed disability has gotten worse</li>
+                <li>VA medical and hospital records that show your rated disability has gotten worse</li>
+                <li>Private medical and hospital records that show your rated disability has gotten worse</li>
               </ul>
-              <p>In some situations you may need to turn in additional forms to support your claim. For example, you’ll need to fill out another form if you’re claiming a dependent or requesting aid and attendance benefits. <a href="#">Learn more about what additional forms you may need to file with your disability claim</a>.</p>
+              <p>In some cases, you may need to turn in one or more supplemental forms to support your claim. For example, you’ll need to fill out another form if you’re claiming a dependent or applying for aid and attendance benefits.<br/> <a href="https://github.com/department-of-veterans-affairs/vets-website/pull/disability-benefits/apply/supplemental-forms/">Learn what additional forms you may need to file with your disability claim</a>.</p>
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx]: Is the non-functional link intentional here? If so, I'd vote for leaving a blank link to make it more obvious or adding a TODO

File: src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -63,7 +76,7 @@ class ConfirmationPage extends React.Component {
             <strong>Conditions claimed</strong>
             <br/>
             <ul className="disability-list">
-              {disabilities.map((disability, i) => {
+              {disabilities.filter(item => item['view:selected']).map((disability, i) => {
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx]: Good catch!

**Review by markgreenburg — APPROVED** [review]
LGTM!

File: src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx (hunk 1 above)
Comment (cehsu) [reply src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx]: Great question @markgreenburg, @peggygannon do we have a link we can provide here?

Comment (peggygannon) [reply src/applications/disability-benefits/526EZ/components/IntroductionPage.jsx]: @cehsu @markgreenburg Yes, the link should be: 
/disability-benefits/apply/supplemental-forms/

this page/link is not live yet. This page will launch when we launch the 526 MVP.

**Comment by peggygannon** [comment]
@cehsu             Hi, claire, thanks for the updated screenshots. I have just one more content edit for the Introduction page: 


Introduction page: 
1.  the link: [Learn what additional forms you may need to file with your claim]   is an internal link, so we need to remove the icon that shows it's an external link. 
the link goes to: disability-benefits/apply/supplemental-forms/

--Please note, this page isn't live yet and will release when the 526 MVP launches.

**Comment by cehsu** [comment]
Thanks @peggygannon, kindly refer to this updated screenshot for that change:
screen shot 2018-05-23 at 5 25 03 pm

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
@cehsu Thanks, Claire. Looks good.

**Pull request merged**